tests/regressions.pl

```diff
@@ -14,8 +14,9 @@
 	my $actual = $_; $actual =~ s/\.sil$/\.actual/;
   my ($unsupported, $knownbad);
   if (-f $expectation) {
-		# Entirely skip tests designed for an OS that is not us
-		if (system("head -n1 $_ | grep -q -P -v 'OS=(?!$^O)'")) {
+        open my $exp, $expectation or die $!;
```

Comment:
Indentation looks wrong - it's gone from tabs to 4 spaces.

Reply:
Oops, I should have caught that.

@simoncozens I don't care for `perl` and you are the expert, tabs or spaces? 2 or 4? Also do you have an EditorConfig extension installed in your editor of choice?

Reply:
4 spaces is normal for Perl. And yes, I have Editorconfig now.

Reply:
Cool beans. This code review comment is tackled in #697.